src/screens/SignTransactionSheet.tsx

```diff
@@ -2108,7 +2111,10 @@ const SimulatedEventRow = ({
   if (asset?.type === TransactionAssetType.Native) {
     assetCode = ETH_ADDRESS;
   }
-
+  const showUSD = (eventType === 'send' || eventType === 'receive') && price;
+  const formattedPrice = `$${price?.toLocaleString?.('en-US', {
+    maximumFractionDigits: 2,
+  })}`;
```

Comment:
Not a blocker on this PR at all, but wonder if it's worth opening up a backlog to explore instances where we display currency and match it to the user's desired currency that they chose in settings.

Seems like in a lot of places we assume USD

Reply:
yeah i think if we're not doing USD we'd keep currency pref on backend and strip all locale conversions on client